files/build_templates/per_namespace/database.service.j2

```diff
@@ -3,6 +3,9 @@ Description=Database container
 {% if multi_instance == 'true' %}
 Requires=database.service
 After=database.service
+{% else %}
```

Comment:
Can we have a variable for chassis_system and then use it here?

Reply:
Either chassis_system or switch type voq will need database-global.service

Reply:
switch type is not published in make file, I don't have clue how we will generate it at compiling time. when it is ready, we use it here, for now, database.service is started by condition, it won't run without globaldb.conf present.